canonical-docs/phone-docs
# moved all assets to the asset.u.c server

## Done

* uploaded all the assets to the asset.ubuntu.com server
* removed all the assets from the site
* replaced all the asset urls with the new asset server ones

## QA

* build the site with documentation builder
* see that the pages look the same

Fixes https://github.com/canonical-websites/docs.ubuntu.com/issues/107
